### PR TITLE
Update .travis.yml to run XCode8

### PR DESCRIPTION
**Pull request by Lucashuang0802**
## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I have resolved any merge conflicts.
- [x] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines), and reviewed the [contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md). Confirmation: @Lucashuang0802

## What's in this pull request?
Update .travis.yml to run XCode8

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -27,6 +27,7 @@ env:
    - DESTINATION="OS=9.2,name=iPad Air 2"   SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  POD_LINT="NO" RUN_UI_TESTS="NO"
    - DESTINATION="OS=9.3,name=iPad Pro"     SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="NO" RUN_UI_TESTS="NO"
 
+   - DESTINATION="OS=10.0,name=iPhone SE"   SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" POD_LINT="NO" RUN_UI_TESTS="NO"
```
Comment (jessesquires) [inline .travis.yml]: let's make this iPhone 7

**Comment by jessesquires** [comment]
oh actually, github let's you make changes in-line now. 😄

**Comment by jessesquires** [comment]
let's wait to merge to see what travis does

**Comment by Lucashuang0802** [comment]
Hmm, it's weird that iPhone 7 is not supported? https://docs.travis-ci.com/user/languages/objective-c/#iOS-10.0

**Comment by jessesquires** [comment]
huh. weird

**Comment by jessesquires** [comment]
this last commit should do it

**Comment by Lucashuang0802** [comment]
nice! @jessesquires gonna rebase and see whether it's working again.

**Pull request merged**